core/src/main/java/io/questdb/cutlass/pgwire/modern/PGConnectionContextModern.java

```diff
@@ -143,6 +143,7 @@ public class PGConnectionContextModern extends IOContext<PGConnectionContextMode
     private static final int PREFIXED_MESSAGE_HEADER_LEN = 5;
     private static final int PROTOCOL_TAIL_COMMAND_LENGTH = 64;
     private static final int SSL_REQUEST = 80877103;
+    private static final long TIMEOUT_MILLIS = 5000; // 5 seconds timeout
```

Comment:
Would be great to introduce a prop for this.

Reply:
It has been deliberately encapsulated.
I'm trying to avoid cluttering the configuration with properties that we are unlikely to change.

Reply:
OK, let's not add a dedicated prop.